### PR TITLE
Benchmarks on CI

### PR DESCRIPTION
**Pull request by DanNixon**

### CODE REVIEW

File: conda/build.sh [hunk 1]
```diff
@@ -28,5 +32,20 @@ ninja -v all-tests install
 ./dataset/test/scipp-dataset-test
 ./neutron/test/scipp-neutron-test
 
+# Build and run benchmarks if they were requested
```
Comment (SimonHeybrock) [inline conda/build.sh:35]: I suppose an intermediate solution would be to just build them (prevent further broken things), but avoid running, to save the time and memory.

**Comment by SimonHeybrock** [comment]
Why closed?

**Comment by DanNixon** [comment]
This is a bit of a dead end approach and a better solution may be available when we move away from Conda for some builds.

**Comment by SimonHeybrock** [comment]
Could you elaborate what the dead end is?

**Comment by DanNixon** [comment]
This works for limiting operations (do nothing or build + run benchmarks, but not just build), I could add more environment variables, but that would start getting messy.

We'd want to run benchmarks outside of a Conda build if we get a separate worker to devote to benchmarks.

**Pull request closed without merging**